src/components/Calendar/EventListSideBar.js

```diff
@@ -1,84 +1,46 @@
-import { events } from './data';
-import { Menu, Dropdown } from 'antd';
-
+import React, { useState } from 'react';
+import Moment from 'moment';
 
 import SideBarEvent from './SideBarEvent';
 
-function EventListSideBar({ showModal, setClickSelected, setSelectLocation }) {
-  const pendings = events
-    .filter((e) => {
-      if (
-        e.eventStatus === false &&
-        e.start.getMonth() + e.start.getDate() ===
-        new Date().getUTCMonth() + new Date().getUTCDate()
-      ) {
-        {
-          return e;
-        }
-      }
-    })
+function EventListSideBar({ events, showModal, setClickSelected }) {
+  const isScheduledToday = (e) => (Moment(e.start).isSame(Moment(Date.now()), 'day'));
+  // eslint-disable-next-line no-unused-vars
+  const [forceRenderHackVar, setForceRenderHackVar] = useState(false);
+  const forceRender = () => {
+    setForceRenderHackVar((oldValue) => !oldValue);
+  };
+  const todaysEvents = events.filter(isScheduledToday);
+
+  const pendingEvents = todaysEvents
+    .filter((e) => e.eventStatus === false)
     .sort((a, b) => new Date(a.start) - new Date(b.start));
-
-  const checkedIns = events.filter((e) => {
-    if (
-      e.eventStatus === true &&
-      e.start.getMonth() + e.start.getDate() ===
-      new Date().getUTCMonth() + new Date().getUTCDate()
-    ) {
-      {
-        return e;
-      }
-    }
-  });
-
-  //change handler for select location
-  const handleLocationChange = (e) => {
-    setSelectLocation(e.item.props.value)
-  }
-
-  const schoolMenu = (
-    <Menu onClick={handleLocationChange}>
-      <Menu.Item value="India" key="1">India</Menu.Item>
-      <Menu.Item value="Africa" key="2">Africa</Menu.Item>
-      <Menu.Item value="AllLocations" key="3">All Locations</Menu.Item>
-    </Menu>
+  const checkedInEvents = todaysEvents.filter((e) => e.eventStatus === true);
+
+  const buildSideBarEvent = (event) => (
+    <SideBarEvent
+      key={event.resourceId}
+      setClickSelected={setClickSelected}
+      showModal={showModal}
+      event={event}
+      forceRender={forceRender}
+    />
   );
 
   return (
     <div className="sidebar-container">
-      <div className="calendar-container">
-        <div className="rbc-toolbar rbc-btn-group">
-          <Dropdown overlay={schoolMenu}>
```

Comment:
We need to have the dropdown, so we can filter the events. Carlos' and other future pull requests are building off this feature, so we may run into some issues merging.

Reply:
added back in

Reply:
Great, Thanks! Should be good now for the other merges!